README.md

```diff
@@ -82,6 +78,22 @@ To migrate from S3 to local storage, the identified steps will be:
     `bin/rake cache:clear`
 5. Restart the Rails server
 
+#### Upgrades: Upgrade modules
+
+Upgrades the gems with engines in them. All, Decidim modules and standard Rails engines.
+
+TO-DO To be finished
+
+#### Upgrades: Validate migrations
+
+Validates that migrations from all gems in the Gemfile have already been installed.
+
+```
+bin/rake cdtb:upgrades:validate_migrations
+```
+
+See the [Installation](#installation) chapter to install a GitHub Action on your app that will run this validation on your CI.
+
```

Comment:
What do you think about unify these sections in one big section named "Upgrades"?

Reply:
I think it would be a little bit more readable after unifying it to "Upgrades" and then with two subsections "Upgrade modules" and "Validate migrations"

Reply:
:ok_hand: